app/controllers/concerns/page_tool_tip.rb

```diff
@@ -0,0 +1,22 @@
+module PageToolTip
+  def set(collection, model_name)
```

Comment:
I agree with renaming it to CollectionPageSetup, I would make necessary adjustments.